include/openssl/cryptoerr_legacy.h

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
This file should be included from all the xxxerr.h files from libcrypto.

Reply:
Yes, you're right.  I thought I had, but that may have gotten lost while I was juggling between this PR and #13392

Reply:
Fixed